DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -40,7 +40,8 @@ ELSE cc.CategoryName
                         au.IsWorkforceContributor,
                         au.IsLocalWorkforceManager,
                         au.ImportOnly,
-                        au.FailedLoginCount
+                        au.FailedLoginCount,
+                        au.Active

```

Comment:
The addition of retrieving Active makes sense, but it has had some knock-on effects on the existing unit tests. It's always a good idea to run all the unit tests to make sure you haven't accidentally broken some functionality (this is one of the developer guidelines found in the checkboxes in the merge request template). 

In this case there are a couple of unit tests that use this method and now fail. The reason is this method is now populating the new Active field which defaults to false, so the returned value (true) does not match the expected value (false). Both of the failing unit tests generate the expected value from a test helper method that has a bunch of optional parameters. I'd recommend adding a new option parameter for Active that defaults to false, but when called in those failing unit tests it passes in the correct value.

On a side note, we also have Jenkins automatically build and run the tests when changes are pushed to our branches - you can see the successes/failures in the hee-notifications channel in slack. Normally, when a run fails in Jenkins you should get notified by an email and slack bot message, but it looks like you might not have been added to the list of users. I will see about getting you added.

Reply:
Activate is removed as we don't need it.